end-to-end-tests/cypress/support/commands.js

```diff
@@ -23,54 +23,57 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
-import '@testing-library/cypress/add-commands';
+import "@testing-library/cypress/add-commands";
 
-Cypress.Commands.add('navToViewer', (viewerLinkText, titleText) => {
-  cy.visit(Cypress.env('base_url'))
-  cy.findByText('OSCAL Catalog Editor').should('exist')
-  cy.get('button').first().click()
-  cy.contains(viewerLinkText).click()
-  cy.findByText(titleText).should('be.visible')
-})
+Cypress.Commands.add("navToViewer", (viewerLinkText, navigationProfile) => {
+  cy.visit(Cypress.env("base_url"));
+  cy.get("button").first().click();
+  cy.contains(viewerLinkText).children('#class="MuiTreeItem-iconContainer"').click();
```

Comment:
Like here I feel like looking for a specific class of children is not really how a user would use the `Editor`. Why not just wait until the `Catalog` button is clickable, click it, then check for a specific file?